sequencer/worker.go

```diff
@@ -55,20 +56,17 @@ func (w *Worker) AddTxTracker(ctx context.Context, tx *TxTracker) {
 
 		root, err := w.state.GetLastStateRoot(ctx, nil)
 		if err != nil {
-			// TODO: How to manage this
 			log.Errorf("AddTx GetLastStateRoot error: %v", err)
 			return
```

Comment:
```suggestion
			return false
```

Reply:
OMG, why does the IDE even let me compile like that? Good catch!